strongbox-security/strongbox-authentication-registry/src/test/java/org/carlspring/strongbox/authentication/ConfigurableProviderManagerTest.java

```diff
@@ -76,34 +68,38 @@
     private PasswordEncoder passwordEncoder;
 
     @Test
-    public void testApplicationContext() 
+    public void testApplicationContext()
     {
         // Check that we have proper implementation of UserDetailsService injected by Spring
         assertThat(userDetailsService).isInstanceOf(ConfigurableProviderManager.class);
     }
-    
+
     @Test
+    @Disabled
```

Comment:
@sbespalov : Do we have a follow-up task for this to be fixed?
What was the reason to have it `@Disabled`?

Reply:
#1802

Reply:
Thanks! ;)